src/Microsoft.DocAsCode.Build.Engine/Incrementals/BuildVersionInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    public class BuildVersionInfo
+    {
+        /// <summary>
+        /// The name of version.
+        /// </summary>
+        public string Version { get; set; }
```

Comment:
>Version [](start = 22, length = 7)

seems this Version means the content version specified by users rather than internal version for track of incremental build? add concrete comments?